features/edit_articles.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Edit articles
+  As a Journalist/Editor
+  In order to manage my contents and keep them updated
+  I would like to be able to edit articles
+
+  Background:
+    Given  following article exists
+    | title               | content                       |
+    | A Whole New World   | A new fantastic point of view |
+
+  Scenario: User can edit an article
+    Given I am on the "A Whole New World" page
+    When I click "Edit Article" link
+    Then I fill in "Title" with "Do you know the Truth?"
```

Comment:
change this to when.

Reply:
I don't think that would make sense, are you sure?

Reply:
```When I click "edit Article" link
When I fill in "Title" with "Do you know the Truth?"```

That doesn't make sense to say that out loud. Do you have any reasoning for a change like that?

Reply:
We use when for actions we make 
Then is what we expect to happen.
In this case And is probs the best since the line before has When